content/en/getting_started/service_catalog/_index.md

```diff
@@ -0,0 +1,181 @@
+
+The following example represents a `shopping-cart` service from an e-commerce application. It includes important metadata about the service, such as its owning team, languages used, runbook link, and code repository." 
+
+{{< code-block lang="yaml" filename="service.datadog.yaml" collapsible="true" >}}
+schema-version: v2.2
+dd-service: shopping-cart
+team: e-commerce
+application: shopping-app
+tier: "1"
+type: web
+languages:
+  - go
+  - python
+contacts:
+  - type: slack
+    contact: https://yourorg.slack.com/archives/e-commerce
+  - type: email
+    contact: ecommerce@example.com
+  - type: microsoft-teams
+    contact: https://teams.microsoft.com/example
+links:
+  - name: Runbook
+    type: runbook
+    url: http://runbook/shopping-cart
+  - name: Source
+    type: repo
+    provider: github
+    url: https://github.com/shopping-cart
+  - name: Deployment
+    type: repo
+    provider: github
+    url: https://github.com/shopping-cart
+  - name: Config
+    type: repo
+    provider: github
+    url: https://github.com/consul-config/shopping-cart
+  - name: E-Commerce Team
+    type: doc
+    provider: wiki
+    url: https://wiki/ecommerce
+  - name: Shopping Cart Architecture
+    type: doc
+    provider: wiki
+    url: https://wiki/ecommerce/shopping-cart
+  - name: Shopping Cart RFC
+    type: doc
+    provider: google doc
+    url: https://doc.google.com/shopping-cart
+tags:
+  - business-unit:retail
+  - cost-center:engineering
+integrations:
+  pagerduty:
+    service-url: https://www.pagerduty.com/service-directory/PSHOPPINGCART
+  opsgenie:
+    service-url: "https://www.opsgenie.com/service/uuid"
+    region: "US"
+ci-pipeline-fingerprints:
+  - id1
+  - id2
+extensions:
+  additionalProperties:
+    customField1: customValue1
+    customField2: customValue2
+{{< /code-block >}}
+
+You can also use any existing knowledge sources your organization maintains, such as Configuration Management Database (CMDB) tables (through [Datadog’s ServiceNow integration][6]) and spreadsheets, to populate your Service Catalog. Datadog also has an [integration with Backstage][7] that allows you to import any data or services registered here into Datadog directly.
+
+Finally, you can also create entries from `service` tags in other Datadog products like Infrastructure Monitoring and Log Management. 
+
+{{< img src="/getting_started/service_catalog/import_entries.jpeg" alt="Import Entries tab in the Service Catalog setup and configuration section" style="width:90%;" >}}
+
+## Managing metadata in Service Catalog
+
+After you create these initial entries in your Service Catalog, it is important to keep the catalog updated consistently so that it remains effective. Service definition files should exist within your team’s version control to make it easier to reference new deployments and other changes to the services where an update may be required. 
```

Comment:
```suggestion
After you create these initial entries in your Service Catalog, it is important to keep the catalog updated consistently so that it remains effective. Service definition files should exist within your team’s version control to make it easier to reference new deployments and other changes to the services where an update may be required. 
```